spec/javascripts/govuk-component/option-select-spec.js

```diff
@@ -124,12 +135,6 @@ describe('GOVUK.OptionSelect', function() {
       expect($optionSelectHTML.hasClass('js-closed')).toBe(false);
     });
 
-    it ('calls setupHeight()', function(){
```

Comment:
Why delete this test?

Reply:
This test assumes setupHeight gets called when the option select is opened, but that's now not always the case. setupHeight is called on init if the option select is in the expanded state, but not if it's collapsed. I'm writing other tests to fill this gap.

Reply:
Have now put this test back in a modified form.